src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
I understand users can add duplicate imports and PSCi works without any problem. However, according to my understanding, duplicate imports mean nothing, but they are all listed in `:show import`. I still think we do not need to keep the duplicates.

It's actually not about `:r`, but `import X` in PSCi. May I revert it from this PR and open a discussion?